### ISSUE TITLE
reddish keywords vs. IDE errors

### ISSUE THREAD

**Issue by marcwittke** [gaplo917/GapStyle#30]
I really like this scheme, but there is one major issue that keeps me from using it thoroughly: the reddish keywords. Call it "badly trained", but my eye is constantly thinking that the IDE is pointing out an error here. To make things worse, actual errors get lost when scanning the code.

Spot the bad symbol (C# on Rider):
![image](https://user-images.githubusercontent.com/4075651/128501680-d09f821b-aa63-426c-b4e8-c0cffe6152d9.png)

**Comment by gaplo917** [comment]
@marcwittke  Normally, the red color or `ResetAccessFailedCountAsyncBla(user)` is mean missing implementation.

Is that true in your case?

**Comment by marcwittke** [comment]
yes, its an invalid identifier. the method is called `ResetAccessFailedCountAsync(user)`. I provoked it to demonstrate the issue: It's hard to distinguish which _red_ is actually a problem.

**Comment by gaplo917** [comment]
@marcwittke Agree, will change on 4.0 release. Thanks for your opinion.

**Comment by gaplo917** [comment]
At the meantime, please try to configure by yourself with the following instructions.

Under Color Scheme > General > Errors and Warnings 

| Properties | Value |
| --- | --- |
| Unknown symbol | Effect `#BC3F3C` Bold Underscored |

![Screenshot 2021-08-06 at 7 46 03 PM](https://user-images.githubusercontent.com/5468398/128506074-fd12185e-1e32-4cb2-92b9-697cbdd2b92d.png)

**Comment by marcwittke** [comment]
cool, I'll give it a try. Thanks for the headsup
